sdk/core/azure-core/test/ut/pipeline_test.cpp

```diff
@@ -52,6 +52,22 @@ TEST(Pipeline, refrefEmptyPipeline)
       std::invalid_argument);
 }
 
+TEST(Pipeline, attestationConstructor)
```

Comment:
Similar here, rename to avoid a service name in the tests for core.

Also, consistency nit: capitalize?
```suggestion
TEST(Pipeline, PiplineDeprecatedConstructor)
```

Reply:
The casing is consistent with the other pipeline tests. And the constructor in question exists for the attestation service *only* - it literally hard codes the name of the attestation service in the constructor. The name in this case is completely appropriate.

Reply:
Makes sense. Looking more closely, the test after is the odd-one out which gave the wrong impression, so ignore the consistency comment.

https://github.com/Azure/azure-sdk-for-cpp/blob/e501f2a6a1ffed99ae3b7881e5f503bcc12ede29/sdk/core/azure-core/test/ut/pipeline_test.cpp#L71